dotcom-rendering/src/lib/getTagFrontsAdPositions.ts

```diff
@@ -0,0 +1,68 @@
+import type { GroupedTrailsBase } from '../types/tagFront';
+import {
+	getMerchHighPosition,
+	MAX_FRONTS_BANNER_ADS,
+	MAX_FRONTS_MOBILE_ADS,
+} from './getFrontsAdPositions';
+
+const hasAdjacentCommercialContainer = (
+	collectionIndex: number,
+	merchHighPosition: number,
+) => collectionIndex === merchHighPosition;
+
+const isEvenIndex = (_collection: unknown, index: number) => index % 2 === 0;
```

Comment:
I was just wondering whether you could import these from `getFrontsAdPositions.ts` as you'd done for `getMerchHighPosition`, or did you want to intentionally duplicate the definitions here?